stake-pool/js/src/instructions.ts

```diff
@@ -604,69 +653,72 @@ export class StakePoolInstruction {
   }
 
   /**
-   * Decode a deposit stake pool instruction and retrieve the instruction params.
+   * Creates an instruction to create metadata
+   * using the mpl token metadata program for the pool token
    */
-  static decodeDepositStake(instruction: TransactionInstruction): DepositStakeParams {
```

Comment:
Looks like all of these changes got clobbered, maybe you need to rebase?

Reply:
I think that belongs to another PR as well